cypress/integration/tests/developer/controls/stakeholdergroups/crud.test.ts

```diff
@@ -104,12 +105,15 @@ describe("Stakeholder group CRUD operations", { tags: "@tier1" }, () => {
         cy.wait(2000);
         cy.get(tdTag)
             .contains(stakeholdergroup.name)
-            .parent(tdTag)
-            .parent(trTag)
+            .closest(tdTag)
```

Comment:
Let's create a generic method in utils called 
"memberAttached(entity, memberName)"

which will take stakeholdergroupName and stakeholder as parameters and return True if it exists .

This method can be used to check if member is attached or not for both stakeholder and stakeholder groups.